test/driver/error_embedding/test.t/run.t

```diff
@@ -0,0 +1,54 @@
+Keep the error output short in order to avoid different error output between different compiler versions in the subsequent tests
+
+  $ export OCAML_ERROR_STYLE=short
+
+With the `-embed-errors` options, if a PPX raises, the first such exception is caught and packed into an output AST
+
+  $ echo "let _ = [%raise]" > impl.ml
+  $ raiser -embed-errors impl.ml
+  [%%ocaml.error "Raising inside the rewriter"]
+
+The same is true when using the `-as-ppx- mode (note that the error is reported by ocaml itself)
+
+  $ ocaml -ppx 'raiser -as-ppx' impl.ml
+  File "./impl.ml", line 1, characters 8-16:
+  Error: Raising inside the rewriter
+  [2]
+
+Also exceptions raised in a preprocessor get embedded into an AST (while the error from the preprocessor's stderr also gets reported on the driver's stderr)
```

Comment:
Could you make sure lines are no longer than 80 columns? Long lines make reading/reviewing the code more difficult. In the github UI, we have to scroll horizontally to read the full sentence.

Reply:
Ok!